core/internal/cltest/cltest.go

```diff
@@ -1618,6 +1618,14 @@ func AssertCount(t *testing.T, db *gorm.DB, model interface{}, expected int64) {
 	require.Equal(t, expected, count)
 }
 
+func AssertCountSqlx(t *testing.T, db *sqlx.DB, tableName string, expected int64) {
+	t.Helper()
+	var count int64
+	err := db.Get(&count, fmt.Sprintf(`SELECT count(*) FROM %s;`, tableName))
+	require.NoError(t, err)
+	require.Equal(t, expected, count)
+}
```

Comment:
Nice, how about completely replacing the old `AssertCount` with this one? Looks like there isn't that many occurrences left 

```
     1	core/web/jobs_controller_test.go:	cltest.AssertCount(t, app.GetDB(), job.DirectRequestSpec{}, int64(n))
     2	core/internal/features_test.go:		cltest.AssertCountStays(t, app.GetDB(), &pipeline.Run{}, 0)
     3	core/cmd/eth_keys_commands_test.go:	cltest.AssertCount(t, db, ethkey.State{}, 1) // The initial funding key
     4	core/cmd/eth_keys_commands_test.go:	cltest.AssertCount(t, db, ethkey.State{}, 3)
     5	core/cmd/eth_keys_commands_test.go:	cltest.AssertCount(t, app.GetDB(), ethkey.State{}, 0)
     6	core/services/bulletprooftxmanager/reaper_test.go:		cltest.AssertCount(t, gdb, bulletprooftxmanager.EthTx{}, 1)
     7	core/services/bulletprooftxmanager/reaper_test.go:		cltest.AssertCount(t, gdb, bulletprooftxmanager.EthTx{}, 1)
     8	core/services/bulletprooftxmanager/reaper_test.go:		cltest.AssertCount(t, gdb, bulletprooftxmanager.EthTx{}, 1)
     9	core/services/bulletprooftxmanager/reaper_test.go:		cltest.AssertCount(t, gdb, bulletprooftxmanager.EthTx{}, 1)
    10	core/services/bulletprooftxmanager/reaper_test.go:		cltest.AssertCount(t, gdb, bulletprooftxmanager.EthTx{}, 0)
    11	core/services/bulletprooftxmanager/reaper_test.go:		cltest.AssertCount(t, gdb, bulletprooftxmanager.EthTx{}, 1)
    12	core/services/bulletprooftxmanager/reaper_test.go:		cltest.AssertCount(t, gdb, bulletprooftxmanager.EthTx{}, 0)
    13	core/services/keeper/upkeep_executer_test.go:	cltest.AssertCountStays(t, db, bulletprooftxmanager.EthTx{}, 0)
    14	core/services/job/job_orm_test.go:		cltest.AssertCount(t, gdb, job.ExternalInitiatorWebhookSpec{}, 2)
    15	core/services/job/job_orm_test.go:		cltest.AssertCount(t, gdb, job.OffchainReportingOracleSpec{}, 1)
    16	core/services/job/job_orm_test.go:		cltest.AssertCount(t, gdb, pipeline.Spec{}, 1)
    17	core/services/job/job_orm_test.go:		cltest.AssertCount(t, gdb, job.OffchainReportingOracleSpec{}, 0)
    18	core/services/job/job_orm_test.go:		cltest.AssertCount(t, gdb, pipeline.Spec{}, 0)
    19	core/services/job/job_orm_test.go:		cltest.AssertCount(t, gdb, job.Job{}, 0)
    20	core/services/job/job_orm_test.go:		cltest.AssertCount(t, gdb, job.KeeperSpec{}, 1)
    21	core/services/job/job_orm_test.go:		cltest.AssertCount(t, gdb, keeper.Registry{}, 1)
    22	core/services/job/job_orm_test.go:		cltest.AssertCount(t, gdb, keeper.UpkeepRegistration{}, 1)
    23	core/services/job/job_orm_test.go:		cltest.AssertCount(t, gdb, job.KeeperSpec{}, 0)
    24	core/services/job/job_orm_test.go:		cltest.AssertCount(t, gdb, keeper.Registry{}, 0)
    25	core/services/job/job_orm_test.go:		cltest.AssertCount(t, gdb, keeper.UpkeepRegistration{}, 0)
    26	core/services/job/job_orm_test.go:		cltest.AssertCount(t, gdb, job.Job{}, 0)
    27	core/services/job/job_orm_test.go:		cltest.AssertCount(t, gdb, job.VRFSpec{}, 0)
    28	core/services/job/job_orm_test.go:		cltest.AssertCount(t, gdb, job.Job{}, 0)
    29	core/services/job/job_orm_test.go:		cltest.AssertCount(t, gdb, job.WebhookSpec{}, 0)
    30	core/services/job/job_orm_test.go:		cltest.AssertCount(t, gdb, job.ExternalInitiatorWebhookSpec{}, 0)
    31	core/services/job/job_orm_test.go:		cltest.AssertCount(t, gdb, job.Job{}, 0)
    32	core/services/keeper/registry_synchronizer_test.go:	cltest.AssertCount(t, db, keeper.Registry{}, 1)
    33	core/services/keeper/registry_synchronizer_test.go:	cltest.AssertCount(t, db, keeper.UpkeepRegistration{}, 2)
    34	core/services/keeper/registry_synchronizer_test.go:	cltest.AssertCount(t, db, keeper.Registry{}, 1)
    35	core/services/keeper/registry_synchronizer_test.go:	cltest.AssertCount(t, db, keeper.UpkeepRegistration{}, 2)
    36	core/services/keeper/registry_synchronizer_test.go:	cltest.AssertCount(t, db, keeper.Registry{}, 1)
    37	core/services/keeper/registry_synchronizer_test.go:	cltest.AssertCount(t, db, keeper.Registry{}, 1)
    38	core/services/keystore/master_test.go:		cltest.AssertCount(t, gdb, keystore.ExportedEncryptedKeyRing{}, 1)
    39	core/services/keystore/master_test.go:		cltest.AssertCount(t, gdb, keystore.ExportedEncryptedKeyRing{}, 1)
    40	core/services/keystore/master_test.go:		cltest.AssertCount(t, gdb, keystore.ExportedEncryptedKeyRing{}, 1)
    41	core/services/keystore/master_test.go:		cltest.AssertCount(t, gdb, keystore.ExportedEncryptedKeyRing{}, 1)
    42	core/services/keystore/master_test.go:		cltest.AssertCount(t, gdb, keystore.ExportedEncryptedKeyRing{}, 1)
    43	core/services/keeper/orm_test.go:	cltest.AssertCount(t, db, &keeper.UpkeepRegistration{}, 1)
    44	core/services/keeper/orm_test.go:	cltest.AssertCount(t, db, &keeper.UpkeepRegistration{}, 1)
    45	core/services/keeper/orm_test.go:	cltest.AssertCount(t, db, &keeper.UpkeepRegistration{}, 3)
    46	core/services/keeper/orm_test.go:	cltest.AssertCount(t, db, &keeper.UpkeepRegistration{}, 1)
    47	core/services/keeper/orm_test.go:	cltest.AssertCount(t, db, &keeper.UpkeepRegistration{}, 5)
    48	core/services/keeper/orm_test.go:	cltest.AssertCount(t, db, &keeper.UpkeepRegistration{}, 3)
    49	core/services/keeper/orm_test.go:	cltest.AssertCount(t, db, keeper.Registry{}, 1)
    50	core/services/keeper/orm_test.go:	cltest.AssertCount(t, db, &keeper.UpkeepRegistration{}, 1)
    51	core/services/keeper/orm_test.go:	cltest.AssertCount(t, db, keeper.Registry{}, 1)
    52	core/services/keeper/orm_test.go:	cltest.AssertCount(t, db, &keeper.UpkeepRegistration{}, 1000)
    53	core/services/keeper/orm_test.go:	cltest.AssertCount(t, db, keeper.Registry{}, 2)
    54	core/services/keeper/orm_test.go:	cltest.AssertCount(t, db, &keeper.UpkeepRegistration{}, 2)
    55	core/services/bulletprooftxmanager/bulletprooftxmanager_test.go:		cltest.AssertCount(t, gdb, bulletprooftxmanager.EthTx{}, 1)
    56	core/services/keystore/p2p_test.go:		cltest.AssertCount(t, gdb, offchainreporting.P2PPeer{}, 2)
    57	core/services/keystore/p2p_test.go:		cltest.AssertCount(t, gdb, offchainreporting.P2PPeer{}, 1)
    58	core/services/bulletprooftxmanager/strategies_test.go:		cltest.AssertCount(t, db, &bulletprooftxmanager.EthTx{}, 7)
    59	core/services/bulletprooftxmanager/nonce_syncer_test.go:		cltest.AssertCount(t, gdb, bulletprooftxmanager.EthTx{}, 0)
    60	core/services/bulletprooftxmanager/nonce_syncer_test.go:		cltest.AssertCount(t, gdb, bulletprooftxmanager.EthTxAttempt{}, 0)
    61	core/services/bulletprooftxmanager/nonce_syncer_test.go:		cltest.AssertCount(t, gdb, bulletprooftxmanager.EthTx{}, 0)
    62	core/services/bulletprooftxmanager/nonce_syncer_test.go:		cltest.AssertCount(t, gdb, bulletprooftxmanager.EthTxAttempt{}, 0)
    63	core/services/bulletprooftxmanager/nonce_syncer_test.go:		cltest.AssertCount(t, gdb, bulletprooftxmanager.EthTx{}, 0)
    64	core/services/bulletprooftxmanager/nonce_syncer_test.go:		cltest.AssertCount(t, gdb, bulletprooftxmanager.EthTxAttempt{}, 0)
    65	core/services/keystore/eth_test.go:		cltest.AssertCount(t, gdb, ethkey.State{}, 1)
    66	core/services/keystore/eth_test.go:		cltest.AssertCount(t, gdb, ethkey.State{}, 0)
    67	core/services/keystore/eth_test.go:		cltest.AssertCount(t, gdb, ethkey.State{}, 2)

```

Reply:
That is the plan. @samsondav split the work by folders among himself, me and @RensR, so eventually all gorm and any code related to gorm should be gone. This particular PR focused on services/keystore folder.